source/UberRides/RideRequestButton.swift

```diff
@@ -105,25 +105,31 @@ import CoreLocation
         sizeToFit()
     }
     
-    // add title, image, and sizing configuration
     override func setContent() {
         super.setContent()
         
+        setSizesConfigs()
+        addTitle()
+        addImage()
+    }
+    
+    private func setSizesConfigs() {
         uberTitleLabel.font = UIFont.systemFontOfSize(17)
         uberTitleLabel.numberOfLines = 2;
         uberTitleLabel.textAlignment = .Right
-        
-        // Add title label
+    }
+    
+    private func addTitle() {
         let titleText = LocalizationUtil.localizedString(forKey: "Ride there with Uber", comment: "Request button description")
         uberTitleLabel.text = titleText
-        
-        // add image
+    }
+    
+    private func addImage() {
```

Comment:
Same idea here, `configureImage()` since we aren't actually adding the image to the view